pom.xml

```diff
@@ -108,13 +108,13 @@
         <compiler.fork>false</compiler.fork>
 
         <!-- Spring-Boot target version -->
-        <spring-boot-version>2.6.6</spring-boot-version>
+        <spring-boot-version>2.7.0</spring-boot-version>
```

Comment:
https://github.com/jboss-fuse/camel-spring-boot/blob/camel-spring-boot-3.14.2-branch/archetypes/camel-archetype-spring-boot/pom.xml#L37 should be updated too

Reply:
@Croway I think we can remove that property from camel-archetype-spring-boot - the archetype inherits from spring-boot-parent which inherits from spring-boot, which has the spring-boot-version property.    Checked the help:effective-pom output and I think removing it would be okay.